app/build.gradle

```diff
@@ -226,6 +227,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     }
 
+    androidTestImplementation Deps.tools_test_okhttp3
```

Comment:
You are defining this dependency twice here? :)

Reply:
done, thanks!